packages/ajax/package.json

```diff
@@ -32,9 +32,11 @@
   "sideEffects": false,
   "dependencies": {
     "@bundled-es-modules/axios": "0.18.1",
-    "@lion/core": "0.13.6",
```

Comment:
This may be correct, because if you extend only `@lion/ajax` and its demos, you need the installation of `@lion/core` for the demo extension to function properly, and it only gets installed if it's a real dependency.